dandi/cli/cmd_organize.py

```diff
@@ -34,6 +34,22 @@
     default="auto",
     show_default=True,
 )
+@click.option(
+    "--modify-external-file-fields",
+    is_flag=True,
+    default=False,
+    help="using this options will rewrite the 'external_file' argument of the "
```

Comment:
Suggested rewording: "Rewrite the 'external_file' arguments of ImageSeries in NWB files.  The new values will correspond to the new locations of the video files after being organized.  This option requires --files-mode to be 'copy' or 'move'."

Reply:
thanks, changes here: https://github.com/dandi/dandi-cli/pull/841/commits/4241edca5a9a434ab9d75a071b60c89af8f34bb5